### PR TITLE
Add interface addressess to domain

### PR DESCRIPTION
**Pull request by smogork**
In this PR is implemented:
1. virDomainInterfacesAddresses from native code.
2. Interpretation of returned data.
3. method on Domain to aqcuire informations about addresses of interfaces inside domain.

### CODE REVIEW

File: src/Virtualization/Libvirt/Native/NativeVirDomain.cs [hunk 1]
```diff
@@ -423,6 +425,58 @@ public static int InterfaceStats(IntPtr dom, string path, out VirDomainInterface
...
+                    Marshal.PtrToStructure(addrPtr, addr);
+                    addresses[j] = addr;
+                }
+
+                ifaces[i] = iface;
+            }
+
+            return ret;
+        }*/
+
```
Comment (smogork) [inline src/Virtualization/Libvirt/Native/NativeVirDomain.cs]: Delete it!

File: src/IDNT.AppBasics.Virtualization.Libvirt.csproj [hunk 2]
```diff
@@ -26,6 +26,7 @@
     <Win32Resource />
     <Platforms>AnyCPU;x64</Platforms>
     <RepositoryType>nuget</RepositoryType>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```
Comment (smogork) [inline src/IDNT.AppBasics.Virtualization.Libvirt.csproj]: It is uneccessary

**Review by smogork — COMMENTED** [review]
@VenMolom Dont merge it yet

**Comment by smogork** [comment]
@VenMolom Its clean now

**Review by VenMolom — APPROVED** [review]

**Pull request merged**